pyvista/plotting/renderer.py

```diff
@@ -1018,7 +1066,6 @@ def add_orientation_widget(
     def add_axes(
         self,
         interactive=None,
-        line_width=2,
```

Comment:
Similarly, please make sure you gracefully handle the deprecation of this keyword argument